modules/programs/bash.nix

```
@@ -11,6 +11,14 @@ in
 {
   meta.maintainers = [ maintainers.rycee ];
 
+  imports = [
+    (mkRenamedOptionModule [ "programs" "bash" "enableAutojump" ] [
+      "programs"
+      "autojump"
+      "enableBashIntegration"
```

To produce equivalent behavior I believe this should be

```suggestion
      "enable"
```

---

Is this because we want to take the highest-level option that is required for equivalent functionality, so to speak?
e.g. since `programs.autojump.enableBaseIntegration` will implicitly be enabled when `programs.autojump.enable` is enabled,  and furthermore, _requires_ the latter is enabled to function at all, we should simply use `programs.autojump.enable` in this case?